asminc/atari.inc

```diff
@@ -997,12 +997,30 @@ diopp_size   = 5        ; size of structure
 ; VALUES for dos_type
 ;-------------------------------------------------------------------------
 
-ATARIDOS     = 0
-SPARTADOS    = 1
-OSADOS       = 2        ; OS/A+
-MYDOS        = 3
-XDOS         = 4
+SPARTADOS    = 0
+OSADOS       = 1        ; OS/A+
+XDOS         = 2
+ATARIDOS     = 3
+MYDOS        = 4
 NODOS        = 255
+; The DOSes with dos_type below or equal MAX_DOS_WITH_CMDLINE do support
+; command line arguments.
+MAX_DOS_WITH_CMDLINE = XDOS
+
+;-------------------------------------------------------------------------
+; XDOS defines (version 2.4, taken from xdos24.pdf)
+;-------------------------------------------------------------------------
+
+XOPT         = $70B     ; XDOS options
+XCAR         = $70C     ; XDOS cartridge address (+ $70D)
+XPAT         = $86F     ; XDOS bugfix and patch number
+XVER         = $870     ; XDOS version number
+XFILE        = $87D     ; XDOS filename buffer
+XLINE        = $880     ; XDOS DUP input line
+XGLIN        = $871     ; get line
+XSKIP        = $874     ; skip parameter
+XMOVE        = $877     ; move filename
+XGNUM        = $87A     ; get number
```

Comment:
> Where is the official "style guide"?

- I presume you asked this question presuming there is none ;-) However there actually is one: https://github.com/cc65/wiki/wiki/C-and-Assembly-sources-style-guide
- The style guide is far from complete and doesn't cover what Greg wrote.